lib/sedna/backend/__init__.py

```diff
@@ -27,14 +27,20 @@ def set_backend(estimator=None, config=None):
     if config is None:
         config = BaseConfig()
     use_cuda = False
+    use_ascend = False
     backend_type = os.getenv(
         'BACKEND_TYPE', config.get("backend_type", "UNKNOWN")
     )
     backend_type = str(backend_type).upper()
     device_category = os.getenv(
         'DEVICE_CATEGORY', config.get("device_category", "CPU")
     )
-    if 'CUDA_VISIBLE_DEVICES' in os.environ:
+
+    # NPU>GPU>CPU
+    if device_category == "ASCEND":
```

Comment:
Maybe it's better to change "ASCEND" to "NPU".

Reply:
This is because mindspore takes “ASCEND" as "device_target", is it necessary to modify it?
[https://www.mindspore.cn/docs/api/zh-CN/r1.3/api_python/mindspore.context.html?highlight=Ascend#module-mindspore.context](url)

Reply:
In fact, this variable `device_category` does not affect the framework, only use to guide our contributors to distinguish between processing units, as usually juxtapose NPU with CPU/GPU, "ASCEND" is only provided by huawei.org. 

In the current scenario, if `MindSpore` supports `ASCEND` only, I think it is best not to set the limit here.